draft-ietf-quic-transport.md

```diff
@@ -1160,33 +1160,29 @@ TRANSPORT_PARAMETER_ERROR.
 
-initial_max_stream_id_bidi (0x0002):
-
-: The initial maximum stream ID parameter contains the initial maximum stream
-  number the peer may initiate for bidirectional streams, encoded as an unsigned
-  32-bit integer.  This value MUST be a valid bidirectional stream ID for a
-  peer-initiated stream (that is, the two least significant bits are set to 0 by
-  a server and to 1 by a client).  If an invalid value is provided, the
-  recipient MUST generate a connection error of type TRANSPORT_PARAMETER_ERROR.
-  Setting this parameter is equivalent to sending a MAX_STREAM_ID
-  ({{frame-max-stream-id}}) immediately after completing the handshake.  The
-  maximum bidirectional stream ID is set to 0 if this parameter is absent,
-  preventing the creation of new bidirectional streams until a MAX_STREAM_ID
-  frame is sent.  Note that a default value of 0 does not prevent the
-  cryptographic handshake stream (that is, stream 0) from being used.
+initial_max_streams_bidi (0x0002):
+
+: The initial maximum streams parameter contains the initial maximum number of
+  application-owned bidirectional streams the peer may initiate, encoded as an
+  unsigned 16-bit integer.  If this parameter is absent or zero,
+  application-owned bidirectional streams cannot be created until a
+  MAX_STREAM_ID frame is sent.  Note that a value of 0 does not prevent the
+  cryptographic handshake stream (that is, stream 0) from being used. For
+  example, a value of 0x05 would be translated to a bidirectional stream ID of
+  20 by a client or 17 by a server.  Setting this parameter is equivalent to
+  sending a MAX_STREAM_ID ({{frame-max-stream-id}}) immediately after completing
+  the handshake containing the resulting Stream ID.
 
 initial_max_stream_id_uni (0x0008):
 
-: The initial maximum stream ID parameter contains the initial maximum stream
-  number the peer may initiate for unidirectional streams, encoded as an
-  unsigned 32-bit integer.  The value MUST be a valid unidirectional ID for the
-  recipient (that is, the two least significant bits are set to 2 by a server
-  and to 3 by a client).  If an invalid value is provided, the recipient MUST
-  generate a connection error of type TRANSPORT_PARAMETER_ERROR.  Setting this
-  parameter is equivalent to sending a MAX_STREAM_ID ({{frame-max-stream-id}})
-  immediately after completing the handshake.  The maximum unidirectional stream
-  ID is set to 0 if this parameter is absent, preventing the creation of new
-  unidirectional streams until a MAX_STREAM_ID frame is sent.
+: The initial maximum streams parameter contains the initial maximum number of
+  application-owned unidirectional streams the peer may initiate, encoded as an
+  unsigned 16-bit integer.  If this parameter is absent or zero, unidirectional
+  streams cannot be created until a MAX_STREAM_ID frame is sent.  For example, a
+  value of 0x05 would be translated to a value of 18 by a client or 19 by a
```

Comment:
s/translated to a value/is equivalent to a MAX_STREAM_ID frame containing 18 when received by a client or 19.../

It's not clear what "value" is from context.  The same applies above.